vital/algo/uuid_factory.h

```diff
@@ -21,10 +19,8 @@ namespace vital {
 namespace algo {
 
 /// Abstract base class for creating uuid's
-
-/**
- *
- */
+///
+///
 class VITAL_ALGO_EXPORT uuid_factory
```

Comment:
This could use some cleanup.  Documentation is missing altogether, but not the comment block.  Strange.